elements/pl-order-blocks/pl-order-blocks.mustache

```diff
@@ -1,4 +1,14 @@
 {{#question}}
+<script>
+$(function() {
+  var UUID = "{{uuid}}";
+
+  new window.PLOrderBlocks(UUID, {
+      maxIndent: {{max_indent}}
+  });
+});
+</script>
```

Comment:
Nit: indentation is all over in this file. Let's stick to either 2 or 4 spaces.

Reply:
LOL I think its actually 4 throughout the file except for right here where I pasted from somewhere else and didn't fix it. 

Side note, are we going to have auto-formatting on mustache files too in the future?

Reply:
Possibly, though that gets trickier because things like `maxIndent: {{max_indent}}` isn't something that a JS formatter should touch. If we were to pass options as `JSON.parse("{{options}}")` instead, that _could_ make auto-formatting via Prettier possible. Food for thought!